Views/CreateSeedView.swift

```diff
@@ -31,7 +31,7 @@ class CreateSeedView: UIView, UITextFieldDelegate {
 
   lazy var specieButton: UIButton = {
     let specieButton = UIButton(frame: CGRect.zero)
-    specieButton.setTitle("Abricotier", for: .normal)
+    specieButton.setTitle("PRUNUS_ARMENIACA".localized, for: .normal)
```

Comment:
Why specifically "PRUNUS_ARMENIACA"? Is it just because it's the first of the list?

Reply:
Yes, because it's the first of the list. For the equipments types I did a method to get the first alphabetical type and according locale, I can do the same here

Reply:
That would be great. Especially since this (Prunus Armeniaca) won't be first in every language.

Reply:
I preferred to do this in ui-improvements branch, because ui-improvements has materials' changes. I hope it's not a problem.. Here is the link of the commit:
https://github.com/ekylibre/zero-ios-v3/commit/6d56e97ed64535c294498567a7388f5dd2d2d267

Reply:
Alright. No problem by me, as long as it's adressed. 👍